insights/parsers/snmp.py

```diff
@@ -79,3 +84,59 @@ def parse_content(self, content):
             else:
                 data_id = line_split[1:]
         self.data = snmp_stats
+
+
+@parser('proc_snmp_ipv6')
```

Comment:
@vishwanathjadhav I believe you need to add this spec to `insights.config.specs`.